request.go

```diff
@@ -1202,7 +1203,7 @@ func NewUpdateRequest(space interface{}) *UpdateRequest {
 	req.rtype = iproto.IPROTO_UPDATE
 	req.setSpace(space)
 	req.key = []interface{}{}
-	req.ops = []interface{}{}
+	req.ops = NewOperations()
```

Comment:
Let's not create an empty object here, this leads to unnecessary allocation to the request.

It's better to check `req.ops == nil` during encoding and use []interface{}{} instead if so.

Reply:
Updated.